src/mobiflight.cpp

```diff
@@ -196,6 +202,15 @@ void loop()
 #if MF_DIGIN_MUX_SUPPORT == 1
         timedUpdate(DigInMux::read, &lastUpdate.DigInMux, MF_INMUX_POLL_MS);
 #endif
+
+#if MF_CUSTOMDEVICE_SUPPORT == 1 && defined(MF_CUSTOMDEVICE_HAS_UPDATE)
+#ifdef MF_CUSTOMDEVICE_POLL_MS
+        timedUpdate(CustomDevice::update, &lastUpdate.CustomDevice, MF_CUSTOMDEVICE_POLL_MS);
+#else
+        CustomDevice::update();
+#endif
+#endif
+
         // lcds, outputs, outputshifters, segments do not need update
```

Comment:
Defines within the platformio.ini for custom devices can enable these update() functions. As this is defined in the customer platformio.ini no changes are requires for new ones